classes/PodsAPI.php

```diff
@@ -155,12 +155,16 @@ public function save_wp_object( $object_type, $data, $meta = array(), $strict =
 			$object_type = 'term';
 		}
 
+		$is_meta_object = in_array( $object_type, array( 'post', 'term', 'user', 'comment' ) );
+
 		if ( $sanitized ) {
 			$data = pods_unsanitize( $data );
-			$meta = pods_unsanitize( $meta );
+
+			// Do not unsanitize metadata. This is already done by WP.
+			//$meta = pods_unsanitize( $meta );
```

Comment:
* :no_entry_sign: No space found before comment text; expected "// $meta = pods_unsanitize( $meta );" but found "//$meta = pods_unsanitize( $meta );"
* :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 3542bce908033fa8f0998ecd7f0a2085</sub>